app/assets/stylesheets/components/_markdown-raw.scss

```diff
@@ -0,0 +1,5 @@
+.app-c-markdown-raw {
+  font-family: inherit;
```

Comment:
If this is a component it should be isolated and not dependent on other styles. In our case we need to declare it explicitly and we can use the following mixin `@include govuk-font(19);`

Reply:
Good point - done.